### DISCUSSION TITLE
Why there are multiple icon images in icons folder?

### DISCUSSION THREAD

**Discussion by maqeelqureshi** [infinitered/ignite#2384] [Q&A] [answered]
Ignite is managing 3 different sized images of one icon, what is the purpose of it? If i want to add new icon should i add 3 images of 24, 48 and 72 pixels>

**Comment by frankcalise** [comment, marked as answer]
Hey @maqeelqureshi thanks for your question! This comes from the Developer Guidelines from Apple/Android where it's mentioned under best practices: https://developer.apple.com/design/human-interface-guidelines/foundations/images/#best-practices

This allows for the image to scale across larger devices with higher screen densities.

You could also use a vector image, such as an SVG if you didn't want to provide the three different sizes like you see in the boilerplate.

**Reply by yulolimum** [reply]
Just adding on to @frankcalise answer. We had a similar question which touches on using SVG here: https://github.com/infinitered/ignite/discussions/2298
